tutorials/text_processing/WFST_Tutorial.ipynb

```diff
@@ -0,0 +1,8368 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "Qq1Hz6CKWdwl",
+    "outputId": "3d8f5bd6-f10e-431d-9039-eb88164fbb95"
+   },
+   "outputs": [],
+   "source": [
+    "if 'google.colab' in str(get_ipython()):\n",
+    "  !pip install -q condacolab\n",
+    "  import condacolab\n",
+    "  condacolab.install()"
+   ]
```

Comment:
Could you add the tutorial to this list https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/starthere/tutorials.html ? Use the gitbub link if colab is not supported.
How about putting a warning if 'google.colab' to let the users know that this notebook doesn't support colab? and remove all `!pip install -q condacolab\n"...`

Reply:
Don't we want to run it through colab eventually thought? Or should tutorials just assume being used as notebooks?